### ISSUE TITLE
Server crash

### ISSUE THREAD

**Issue by qSorin** [pawn-lang/YSI-Includes#421]
Hi, I have a problem, since I switched to YSI 5.x my server started to shut down instantly, meaning the players have desycn and they cannot connect to the server and from the closed server.

Sorry for my bad english, used Google Translate!!

**Comment by Y-Less** [comment]
That's really not much to go on unfortunately.

**Comment by qSorin** [comment]
Who?

dum., 5 apr. 2020, 14:55 Y-Less <notifications@github.com> a scris:

**Comment by Y-Less** [comment]
What?

**Comment by qSorin** [comment]
My server crashing on use a YSI 5.x version and i use the YSI 4.x verion, my server don't that crash!

**Comment by Y-Less** [comment]
Yes, I got that, but just saying "it crashes" is not helpful at all.  I need something that can reproduce the crash, an idea of what libraries you are using, or literally anything else to help narrow down the problem from "maybe YSI".

**Comment by qSorin** [comment]
#define YSI_NO_HEAP_MALLOC
#include <YSI_Coding\y_malloc>
#include <YSI_Coding\y_hooks>
#include <YSI_Data\y_iterate>
#include <YSI_Core\y_master>
#include <YSI_Coding\y_timers>

And server working on 20-30 minutes and crashing, why?

**Comment by rt-2** [comment]
I have a similar problem here. My server works for 10-20min then every client gets a message saying the server is restarting constantly.
